.github/workflows/ci_tests_dev.yaml

```
@@ -0,0 +1,79 @@
+# This workflow installs PyGMT dependencies, builds documentation and runs tests on GMT master
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: GMT Master Tests
+
+on:
+  # push:
+  #   branches: [ master ]
+  pull_request:
+    types: [review_requested, ready_for_review]
+  # Schedule daily tests
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  test_gmt_master:
+    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-20.04, macOS-10.15]
+    env:
+      # LD_LIBRARY_PATH: ${{ github.workspace }}/gmt/lib:$LD_LIBRARY_PATH
+      GMT_INSTALL_DIR: ${{ github.workspace }}/gmt-install-dir
+      GMT_LIBRARY_PATH: ${{ github.workspace }}/gmt-install-dir/lib
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      # Checkout current git repository
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
+        with:
+          # fecth all history so that versioneer works
+          fetch-depth: 0
+
+      # Setup Miniconda
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.6.0
+        with:
+          python-version: ${{ matrix.python-version }}
+          channels: conda-forge
+
+      # Install build dependencies from conda-forge
+      - name: Install build dependencies
+        run: conda install cmake libblas libcblas liblapack fftw gdal ghostscript libnetcdf hdf5 zlib curl pcre gshhg-gmt dcw-gmt ipython pytest pytest-cov pytest-mpl
```

No need to install dcw-gmt and gshhg-gmt. The build-gmt-master.sh script downloads these two packages.
```suggestion
        run: conda install cmake libblas libcblas liblapack fftw gdal ghostscript libnetcdf hdf5 zlib curl pcre ipython pytest pytest-cov pytest-mpl
```